src/PIL/Image.py

```diff
@@ -355,10 +355,11 @@ def init():
     if _initialized >= 2:
         return 0
 
+    parrent_module = __name__.replace(".Image", "")
```

Comment:
This will break paths like "Project.ImageLibs.PIL.Image".

```suggestion
    parrent_module, _, _ = __name__.rpartition('.')
```

Reply:
OK, thanks, I'll fix it